components/ui/AppCard.stories.ts

```diff
@@ -0,0 +1,23 @@
+import { Meta, Story } from '../../node_modules/@storybook/vue/types-6-0'
+import AppCard from './AppCard.vue'
+
+export default {
+  title: 'UI/App/Card',
+  component: AppCard
+} as Meta
+
+const Template: Story = (args, { argTypes }) => ({
+  components: { AppCard },
+  props: Object.keys(argTypes),
+  template: '<AppCard :cta-label="ctaLabel" :image="image" :tags="tags" :title="title" :to="to">{{ vSlot }}</AppCard>'
+})
+
+export const Default = Template.bind({})
+Default.args = {
+  ctaLabel: 'Learn more',
+  image: 'https://via.placeholder.com/300',
```

Comment:
(non-blocking)

I'm not sure if this is supposed to load in the Storybook, but it doesn't load for me.

Reply:
Thanks for the heads up!

It doesn't load indeed due to `nuxt-lazy-load`. I will have to look deeper into it when creating the next story that uses it.

I would try and fix it for `AppCard.stories.ts` as well, but this component in particular is already deprecated, so it doesn't make sense to work on that right now.

Since this is not specific to the Storybook setup, I'll leave it as-is for this PR 🙂